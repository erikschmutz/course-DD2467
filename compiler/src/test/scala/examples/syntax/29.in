let add = function a -> a * 2;; 
(* These functions should be the same *)
let add a = a * 2;;