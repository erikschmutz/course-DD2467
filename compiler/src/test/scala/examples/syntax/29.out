Program(
    List(
        Assignment(
            Identifier(add),
            LetBinding(
                Identifier(a),
                OperatorExpr(
                    Identifier(a),
                    IntLit(2),
                    Multiply()
                )
            )
        ),
        Assignment(
            Identifier(add),
            LetBinding(
                Identifier(a),
                OperatorExpr(
                    Identifier(a),
                    IntLit(2),
                    Multiply()
                )
            )
        )
    )
)